addons/sms/wizard/send_sms.py

```diff
@@ -88,7 +88,7 @@ def action_send_sms(self):
         active_model = self.env.context.get('active_model')
         model = self.env[active_model]
         records = self._get_records(model)
-        if getattr(records, 'message_post_send_sms'):
+        if hasattr(records, 'message_post_send_sms'):
```

Comment:
better here is
```python
try:
    getattr()(self.message, numbers=numbers)
except AttributeError:
    self.env['sms.api']._send_sms(numbers, self.message)

Reply:
I have to disagree, since this would mask any `AttributeError` raised by `message_post_send_sms()` and call `_send_sms()` instead.